### PR TITLE
docs: hugo version bump to 0.97.2

### PR DESCRIPTION
**Pull request by eallion**

### CODE REVIEW

**Comment by eallion** [comment]
I sort the index.md by `sort`, but check it fails.

**Comment by eallion** [comment]
The shell script:
https://gist.github.com/eallion/aaecaec8107912755c4a22681687da30

**Review by sinkcup — APPROVED** [review]

File: index.md [hunk 1]
```diff
@@ -13,18 +13,18 @@ dependency-check-release.zip | 6.3.1 | https://github.com/jeremylong/DependencyC
 git-lfs-linux-amd64.tar.gz | 2.11.0 | https://github.com/git-lfs/git-lfs/releases/download/v2.11.0/git-lfs-linux-amd64-v2.11.0.tar.gz | 46508eb932c2ec0003a940f179246708d4ddc2fec439dcacbf20ff9e98b957c9
 go-darwin-amd64.pkg | 1.18 | https://go.dev/dl/go1.18.darwin-amd64.pkg | dbfabbbb6377c225ccf28393ecc7aed55564870fcfb39d797411d6b8693013c6
 go-darwin-arm64.pkg | 1.18 | https://go.dev/dl/go1.18.darwin-arm64.pkg | 7c48130b37a155e709ef1db22d3f40ca17cfdaee0dc9d0cc35d2d7c4e0e3d299
+golangci-lint-linux-amd64.tar.gz | 1.43.0 | https://github.com/golangci/golangci-lint/releases/download/v1.43.0/golangci-lint-1.43.0-linux-amd64.tar.gz | f3515cebec926257da703ba0a2b169e4a322c11dc31a8b4656b50a43e48877f4
 go-linux-amd64.tar.gz | 1.18 | https://go.dev/dl/go1.18.linux-amd64.tar.gz | e85278e98f57cdb150fe8409e6e5df5343ecb13cebf03a5d5ff12bd55a80264f
-golangci-lint-linux-amd64.tar.gz | 1.43.0 | https://github.com/golangci/golangci-lint/releases/download/v1.43.0/golangci-lint-1.43.0-linux-amd64.tar.gz | f3515cebec926257da703ba0a2b169e4a322c11dc31a8b4656b50a43e48877f4 |
```
Comment (sinkcup) [inline index.md:17]: 这个顺序是对的，改了导致 CI 挂了

**Review by sinkcup — CHANGES_REQUESTED** [review]

**Comment by sinkcup** [comment]
> The shell script: https://gist.github.com/eallion/aaecaec8107912755c4a22681687da30

奇怪，我看看

**Pull request closed without merging**